packages/components/src/components/forms/QueryInfoForm.tsx

```diff
@@ -354,7 +353,7 @@ export class QueryInfoForm extends React.Component<QueryInfoFormProps, State> {
                             <Button
                                 className="test-loc-submit-button"
                                 bsStyle="success"
-                                disabled={isSubmitting || !canSubmit || count === 0 || !(canSubmitNotDirty || isDirty)}
+                                disabled={isSubmitting || fieldEnabledCount == 0 || !canSubmit || count === 0 || !(canSubmitNotDirty || isDirty)}
```

Comment:
if canSubmitNotDirty is true, should button be enabled even if all fields are disabled?